### PR TITLE
Atabeg

### PR DESCRIPTION
**Pull request by ATABEG00**
new thing

### CODE REVIEW

File: src/Test/LocalTest.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package Test;
+
+public class LocalTest {
+    public static void main(String[] args) {
+        System.out.println();
```
Comment (Ozleblebici) [inline src/Test/LocalTest.java]: I guess we should add verification for local test!!

**Review by Ozleblebici — COMMENTED** [review]
add 1 comment pls check it

File: src/Test/LocalTest.java (hunk 1 above)
Comment (ATABEG00) [reply src/Test/LocalTest.java]: upgraded. check pls

Comment (Ozleblebici) [reply src/Test/LocalTest.java]: looks nice thanks

**Review by Ozleblebici — APPROVED** [review]
All good for me

**Pull request merged**